test/performance/long-running-tests/workloads/long-duration-exp-workload.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sleep-20-sec
+  labels:
+    app.kubernetes.io/name: perf-test
+spec:
+  containers:
+  - name: aw-wait-20
+    image: ubuntu:latest
+    imagePullPolicy: IfNotPresent
+    command: [ "/bin/bash", "-c", "sleep 20" ]
+  restartPolicy: Never
```

Comment:
I would see less red if this file had a terminal linebreak.

Reply:
Fixed!

Reply:
I still see the red circle.

<img width="904" alt="image" src="https://github.com/user-attachments/assets/05694f87-6699-43d0-8af6-d5222a308a4b" />

Reply:
Oh! I see, I double-checked. It's fixed now.